activerecord/test/schema/schema.rb

```diff
@@ -503,6 +503,18 @@
     t.column :weight, :integer
   end
 
+  create_table(:measurements, id: false, force: true, options: "PARTITION BY LIST (city_id)") do |t|
```

Comment:
This (and following) measurements table definitions in schema are valid only for postgres adapter.

CI can't create those tables on mysql and sqlite.

https://buildkite.com/rails/rails/builds/64610#b993a2df-ea4f-45f4-a6f2-0dd8487e2237
https://buildkite.com/rails/rails/builds/64610#8112f1a1-8b9e-4cf0-9b48-49f8caf5fb24

Any chance to make this table conditional and load coresponding model only when currently used adapter `supports_partitioned_indexes`?

Reply:
Hey @simi, thanks for the feedback. And yes, I'm gonna take a look at that, didn't check before if they were _adapter constrained_.